tests/kafkatest/services/verifiable_producer.py

```diff
@@ -73,24 +73,27 @@ def __init__(self, context, num_nodes, kafka, topic, max_messages=-1, throughput
         self.acked_values = []
         self.not_acked_values = []
         self.produced_count = {}
-        self.prop_file = ""
+
+    def prop_file(self, idx):
```

Comment:
Thanks for the info, I did not know that. Will change the signature to take the node, same as in console consumer.